diff --git a/ros2/rviz/rviz_ogre_vendor/CMakeLists.txt b/ros2/rviz/rviz_ogre_vendor/CMakeLists.txt
index c400c07f..574c74bf 100644
--- a/ros2/rviz/rviz_ogre_vendor/CMakeLists.txt
+++ b/ros2/rviz/rviz_ogre_vendor/CMakeLists.txt
@@ -143,7 +143,7 @@ macro(build_ogre)
       "-DZLIB_ROOT=${CMAKE_CURRENT_BINARY_DIR}/zlib-install")
   endif()
 
-  if(DEFINED CMAKE_TOOLCHAIN_FILE)
+  if(false)
     list(APPEND extra_cmake_args "-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}")
   else()
     list(APPEND extra_cmake_args "-DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}")
