diff --git a/BehaviorTree.CPP/cmake/FindZMQ.cmake b/BehaviorTree.CPP/cmake/FindZMQ.cmake
index c4b8c23..7c3f1f9 100644
--- a/BehaviorTree.CPP/cmake/FindZMQ.cmake
+++ b/BehaviorTree.CPP/cmake/FindZMQ.cmake
@@ -13,10 +13,11 @@
 #  For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 #
 
+find_package(ZeroMQ)
 if(ZeroMQ_FOUND)
   set(ZMQ_FOUND ${ZeroMQ_FOUND})
   set(ZMQ_INCLUDE_DIRS ${ZeroMQ_INCLUDE_DIR})
-  set(ZMQ_LIBRARIES ${ZeroMQ_LIBRARY})
+  set(ZMQ_LIBRARIES libzmq)
 else()
 
 
